netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -190,18 +188,6 @@ public final NettyChannelBuilder flowControlWindow(int flowControlWindow) {
   }
 
   /**
-   * Sets the maximum message size allowed to be received on the channel. If not called,
-   * defaults to 4 MiB. The default provides protection to clients who haven't considered the
-   * possibility of receiving large messages while trying to be large enough to not be hit in normal
-   * usage.
-   */
-  public final NettyChannelBuilder maxMessageSize(int maxMessageSize) {
```

Comment:
Removing this will break people. Mark it as deprecated for now and we remove it after a while?